### ISSUE TITLE
Revisit our flux definition

### ISSUE THREAD

**Issue by rmjarvis** [GalSim-developers/GalSim#159]
Twice recently, there has been some confusion about what our `obj.getFlux()` function really means, so it seems worth revisiting this definition here.

Currently, when an object is drawn to an image, the flux of the object (that was set before the draw() or drawShoot() command) ends up being equal to (Sum I_ij dx dy).  This is correct if what we want drawn on the image is surface brightness: flux / arcsec^2.  However, I don't think this is probably what we want.  I think the "flux" should mean Sum I_ij.  So if we redraw an object using half the pixel scale, each pixel should get a quarter of the flux it had in the first image.

In other words, our GSObject or SBProfile should define a surface brightness profile in terms of flux/arcsec^2.  And the flux that gets set by `obj.setFlux(flux)` should be int I(x,y) dx dy.  But when drawing onto an image, that total flux should end up as Sum I_ij.

What do people think?  Am I totally off base here?
